.sass-lint.yml

```diff
@@ -237,7 +240,7 @@ rules:
 
   # enforce how deeply a selector can be nested (defaults to 2)
   # https://github.com/sasstools/sass-lint/blob/master/docs/rules/nesting-depth.md
-  nesting-depth: 2
+  nesting-depth: 3
```

Comment:
# 0 - disabled
 # 1 - warning
 # 2 - error

Change the max-depth instead if you can for now. Put it to whatever largest depth it complains about. I did this for a number of repos because trying to redo the css is a big job.